src/PropertyInfo/ConstructorExtractor.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\PropertyInfo;
+
+use ReflectionClass;
+use ReflectionException;
+use Symfony\Component\PropertyInfo\PropertyAccessExtractorInterface;
+
+class ConstructorExtractor implements PropertyAccessExtractorInterface
```

Comment:
I don't think so. The fact that a parameter is writable is driven by the fact that you can write it for an _instantiated_ object. So this could never be inside the Symfony component.

Reply:
I tend to agree, this a low level and common need, it should be in Symfony, not in API Platform (history: we moved the code PropertyInfo code from API Platform to a standalone library then to Symfony for this exact same reason).

Reply:
Adding `property_info.initializable_extractor` tag is better than using `property_info.access_extractor`, I like this approach